wsl-builder/common/releasesinfo.go

```diff
@@ -9,21 +14,23 @@ import (
 	"time"
 )
 
-type wslReleaseInfo struct {
-	WslID         string
-	FullName      string
-	BuildVersion  string
-	LauncherName  string
-	ShortVersion  string
-	IconVersion   string
-	ReservedNames []string
-
-	codeName    string
-	shouldBuild bool
+// WslReleaseInfo models information about the Ubuntu releases to be used
+// in the template files which are part of the WSL image and launcher build process.
+type WslReleaseInfo struct {
+	WslID         string   // uniquely identifies this distro in WSL context.
```

Comment:
The nitpick would be here: some comments starts with a capital letter, some not, and one is missing a leading space. Also, some don’t end up with a colon.

I’m sure there are other places when it’s not compliant in the code, so we can do a global cleanup in one shot.